src/app.module.ts

```diff
@@ -16,11 +16,14 @@
 
 import { Module } from '@nestjs/common';
 import { ConfigModule } from '@nestjs/config';
+import { TerminusModule } from "@nestjs/terminus";
 import { TokensModule } from './tokens/tokens.module';
 import { EventStreamModule } from './event-stream/event-stream.module';
 import { EventStreamProxyModule } from './eventstream-proxy/eventstream-proxy.module';
+import {HealthController} from "./health/health.controller";
 
 @Module({
-  imports: [ConfigModule.forRoot(), TokensModule, EventStreamModule, EventStreamProxyModule],
+  imports: [ConfigModule.forRoot(), TokensModule, EventStreamModule, EventStreamProxyModule, TerminusModule],
+  controllers: [HealthController]
```

Comment:
If the HealthController is going to live in a subfolder, I'd prefer it to have its own `health.module.ts` for consistency. Alternately if it's going to be part of the app module, it can just live at the root.

Reply:
This was just me being lazy / still learning nest.js I believe I got it correct now.